packages/calypso-e2e/src/lib/pages/marketing-page.ts

```diff
@@ -51,9 +43,18 @@ export class MarketingPage extends BaseContainer {
 	 * @returns {Promise<void>} No return value.
 	 */
 	async clickTabItem( name: string ): Promise< void > {
+		const navTabs = await this.page.waitForSelector( selectors.navTabs );
+		const isDropdown = await navTabs
+			.getAttribute( 'class' )
+			.then( ( value ) => value?.includes( 'is-dropdown' ) );
 		const sanitizedName = toTitleCase( [ name ] );
 
-		await this.page.click( `text=${ sanitizedName }` );
+		if ( isDropdown ) {
+			await navTabs.click();
+			await this.page.click( `${ selectors.navTabsDropdownOptions } >> text=${ name }` );
```

Comment:
following up on previous comment: 
```suggestion
			await this.page.click( `${ selectors.navTabsDropdownOption } >> text=${ name }` );
```